channeldb/channel.go

```diff
@@ -444,6 +444,10 @@ type OpenChannel struct {
 	// Capacity is the total capacity of this channel.
 	Capacity btcutil.Amount
 
+	// LocalFundedAmt is the amount the local node funded into the target
+	// channel.
+	LocalFundedAmt btcutil.Amount
```

Comment:
Fields cannot be arbitrarly added like this, since they will also need to be supported in serialization.